### PR TITLE
Fix various deprecation warnings

### PR DESCRIPTION
**Pull request by esoergel**
Don't merge, I'd like to throw this on staging for a while (it's up now)

### CODE REVIEW

File: corehq/apps/orgs/views.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def invite(self, invitation, user):
         user.add_org_membership(self.organization)
         user.save()
 
-@transaction.commit_on_success
+@transaction.atomic
```
Comment (dimagimon) [inline corehq/apps/orgs/views.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/registration/views.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def get_domain_context(domain_type='commcare'):
 def registration_default(request):
     return redirect(register_user)
 
-@transaction.commit_on_success
+@transaction.atomic
```
Comment (dimagimon) [inline corehq/apps/registration/views.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/registration/views.py [hunk 3]
```diff
@@ -177,7 +177,7 @@ def register_domain(request, domain_type=None):
     })
     return render(request, 'registration/domain_request.html', context)
 
-@transaction.commit_on_success
+@transaction.atomic
```
Comment (dimagimon) [inline corehq/apps/registration/views.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/registration/views.py [hunk 4]
```diff
@@ -219,7 +219,7 @@ def resend_confirmation(request):
     })
     return render(request, 'registration/confirmation_resend.html', context)
 
-@transaction.commit_on_success
+@transaction.atomic
```
Comment (dimagimon) [inline corehq/apps/registration/views.py]: E302 expected 2 blank lines, found 1

**Comment by esoergel** [comment]
ah well.  I wanted to see if a straight substitution would Just Work.

**Comment by benrudolph** [comment]
did all of these fail on straight substitution? or just a few?

**Comment by esoergel** [comment]
Just a few - the get_queryset business works (was planning on PRing that separately), the commit_on_success stuff didn't work in some cases, but I didn't bother to figure out which, and I saw Simon also had a PR to update Raven.

**Comment by benrudolph** [comment]
cool sounds good. just upgraded to 1.6 and am now feeling the deprecation warnings. let me know if you want to divide and conquer on any of them

**Pull request closed without merging**